packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -312,7 +314,12 @@ export const QueryBar: React.FunctionComponent<QueryBarProps> = ({
           </div>
         )}
       {isAIFeatureEnabled && (
-        <div className={queryAIContainerStyles}>
+        <div
+          className={cx(
+            queryAIContainerStyles,
+            isAIInputVisible && visibleAIContainerStyles
+          )}
```

Comment:
this aligns the two toolbar containers perfectly. when user navigates between tabs, there's no flicker.